lib/async.zsh

```diff
@@ -23,12 +23,17 @@ typeset -g GEOMETRY_ASYNC_PROC_ID=0
     GEOMETRY_ASYNC_PROC_ID=$!
 }
 
-# Hooks precmd to run `-geometry_async_function` and hooks
-# `USR1` signal to comunicate with it
+# Removes the async temp file when zsh exits
+-geometry_async_zshexit() {
+    rm -f "${GEOMETRY_ASYNC_TMP_FULL_PATH}$$"
```

Comment:
I'm with @jedahan on this one. We should force the usage of the actual `rm` as there were similar problems in #63.

Correct me if I'm wrong but you dropped the `/dev/null` redirection since it's now a hook, correct?